src/types.h

```diff
@@ -72,8 +72,32 @@ enum class ProbeType
   hardware,
 };
 
+struct ProbeItem
+{
+  std::string name;
+  std::string abbr;
+  ProbeType type;
+};
+
+const ProbeItem PROBE_LIST[] =
+{
+  { "kprobe", "k", ProbeType::kprobe },
+  { "kretprobe", "kr", ProbeType::kretprobe },
+  { "uprobe", "u", ProbeType::uprobe },
+  { "uretprobe", "ur", ProbeType::uretprobe },
+  { "usdt", "usdt", ProbeType::usdt },
```

Comment:
Suggestions for usdt:

```suggestion
  { "usdt", "ut", ProbeType::usdt },
```

or

```suggestion
  { "usdt", "U", ProbeType::usdt },
```

Reply:
I don't like ut. "U" could be ok, although I think in my scripts I'll stick to usdt.